### ISSUE TITLE
Min Value swapped with Max Value

### ISSUE THREAD

**Issue by AlexZibin** [adafruit/Adafruit_BMP280_Library#47]
I've compiled the example code, everything woks great except that **Min Value** is swapped with **Max Value**.
Here's the output: 
```
BMP280 Sensor event test
------------------------------------
Sensor:       BMP280
Type:         Ambient Temp (C)
Driver Ver:   1
Unique ID:    280
Min Value:    85.00
Max Value:    -40.00
Resolution:   0.01
------------------------------------

Temperature = 23.46 *C
Pressure = 1012.14 hPa
```

**Comment by ladyada** [comment]
woops, can you submit a PR to fix / swap the numbers? :)

**Comment by caternuson** [comment]
See #48 . Looks like same typo was in the pressure characterization also. Fixes both.

**Comment by caternuson** [comment]
Closing. Fixed by #48. Thanks again @AlexZibin for pointing this out.
